github/requesting_review.md

```diff
@@ -0,0 +1,58 @@
+# Requesting review
+
+This document tires to explain who should be requested for review when crafting a PR.
+
+## Default
+
+By default a review **should** be requested from:
+
+- A team. Usually your team or [team owning the component][ownership].
+- Individual stakeholders or person who can bring valuable input.
+
+By default a review **should not** be requested from:
+
+- Chapter or SIG.
+
+**NOTE:** Consider pinging individuals outside the team in comments instead of
+requesting review on the whole change. Maybe their input is needed only for
+a part of the change and there is no reason to involve them in the whole PR.
+
+## Exceptions
+
+For information spreading:
+
+- Request review from relevant Chapters for documentation. E.g. Chapter PE for
+  documentation for creating releases.
+- Request review from relevant Chapters for `fmt`. E.g. Chapter FE for
```

Comment:
I think for PRs in `happa` and `giantswarm-js-client` @giantswarm/chapter-front-end should always be pinged.

Reply:
I'm fine with that. Can you suggest a change?

Reply:
:point_up: